dist/libblockdev.spec.in

```diff
@@ -58,6 +50,23 @@ Requires: glib2-devel
 This package contains header files and pkg-config files needed for development
 with the libblockdev library.
 
+%package -n python2-%{name}
+Summary:     Python2 gobject-introspection bindings for libblockdev
+Requires: %{name}%{?_isa} = %{version}-%{release}
+%{?python_provide:%python_provide python2-%{name}}
```

Comment:
And for what it's worth, python2-whatever instead of python-whatever is the new hotness per this, the fifth update this year to the python packaging guidelines: https://fedoraproject.org/wiki/Packaging:Python